src/Microsoft.AspNet.Http.Abstractions/HttpContext.cs

```diff
@@ -36,6 +36,8 @@ public abstract class HttpContext : IDisposable
 
         public abstract ISession Session { get; set; }
 
+        public abstract string TraceIdentifier { get; set; }
```

Comment:
Happy to back peddle; but want to put a PR on a PR and my git fu is not strong enough...